Monika After Story/game/script-topics.rpy

```diff
@@ -4791,6 +4791,42 @@ label monika_urgent:
     m 1k "I'm sure it's adorable~"
     return
 
+# For George Orwell's Nineteen Eighty-Four, Monika avoids political usage,
+# as it reminds her of the debate club, but more the interpretation of language
+# and thought, which she would feel better connected with.
+# I did not use the 'Big Brother','surveillance' keywords, 
+# because either they may get used on other topics
+	
+init 5 python:
+    for key in ['1984','Nineteen Eighty-Four','Ingsoc','newspeak','George Orwell']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('monika_1984')
+    monika_random_topics.append('monika_1984')
+
+label monika_1984
```

Comment:
your indentation for all the dialogue in this label is wrong. Please make sure that your indentation settings in whatever editor you are using are set to `spaces` and `4`. (We want 4 space indentations).

Reply:
I have hopefully done the proper corrections of capitalization and spaces.